docs/runbooks/SSPOperatorDown.md

```diff
@@ -1,30 +1,43 @@
 # SSPOperatorDown
+<!--apinnick, Nov. 2022-->
 
 ## Meaning
 
-The SSP Operator is responsible for deploying and reconciling the common-templates and the template validator.
-This alert fires when the SSP operator is down.
+This alert fires when the Scheduling, Scale and Performance (SSP) Operator is down.
+
+The SSP Operator is responsible for deploying and reconciling the common templates and the Template Validator.
```

Comment:
@apinnick I believe this line should be added also to this PR https://github.com/kubevirt/monitoring/pull/115/files.

Reply:
good idea